include/barcodes/datamatrix.php

```diff
@@ -470,7 +470,7 @@ protected function get255StateCodeword($cwpad, $cwpos) {
 	 * Returns true if the char belongs to the selected mode
 	 * @param $chr (int) Character (byte) to check.
 	 * @param $mode (int) Current encoding mode.
-	 * @return boolean true if the char is of the selected mode.
+	 * @return bool true if the char is of the selected mode.
```

Comment:
Instead of fixing these barcode functions it would probably be easier to just import the new tc-lib-barcode library that is ready.

Reply:
Would that be a breaking change?